lib/data/network/media/cancel_exception.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:equatable/equatable.dart';
+
+class CancelRequestException with EquatableMixin {
+  final String? reason;
+
+  CancelRequestException({this.reason = ""});
```

Comment:
IMO: No need

Reply:
maybe useful message in case of upload cancel and download cancel

Reply:
It's can be null?